### PR TITLE
docs: add FAQ entry for rebooting and powering off

### LINKED ISSUE #598: 'poweroff' does not exit the VM
Issuing `reboot` exist the VM process (slightly unexpected) while `poweroff -f` or `halt -f` shuts down but does not exit the VM process.

### PR DESCRIPTION
**Pull request by acatangiu**
Fixes #598

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -151,6 +151,21 @@ mapping:
 /network-interfaces/0 -> eth1
 ```
 
+### How can I gracefully reboot the guest? How can I gracefully poweroff the guest?
```
Comment (andreeaflorescu) [inline FAQ.md]: `mdl` is reporting the following error:
FAQ.md:154: MD026 Trailing punctuation in header

Otherwise LGTM.

Comment (acatangiu) [reply FAQ.md]: What does that mean though?

Comment (acatangiu) [reply FAQ.md]: Looking at https://github.com/markdownlint/markdownlint/issues/72 it seems like `?` is by default disallowed punctuation in headers.

I believe for the FAQ it is totally normal to have headers as Frequently Asked **Questions**, so I propose we ignore the error.

Comment (andreeaflorescu) [reply FAQ.md]: Ok.

**Pull request merged**